### PR TITLE
Reader: add Conversations empty state illustration

### PR DESCRIPTION
**Pull request by bluefuton**
## Reader empty state:
Shown when user has not Liked or Commented on any post.
empty-state-mockup001

Fixes https://github.com/Automattic/wp-calypso/issues/17184.

### CODE REVIEW

**Comment by bluefuton** [comment]
@jancavan I've added the new empty component. Here's how it looks with the standard `EmptyContent` styles:

screen shot 2017-10-10 at 15 28 01

Do we want to do some tweaking to match the screenshot above?

File: client/reader/stream/empty.jsx [hunk 1]
```diff
@@ -24,15 +23,15 @@ class FollowingEmptyContent extends React.Component {
 	};
 
 	render() {
-		const action = isDiscoverEnabled() ? (
```
Comment (bluefuton) [inline client/reader/stream/empty.jsx]: Quick fix: I noticed that this button shouldn't be gated on Discover being enabled. It must have pointed there originally but now sends the user to `/read/search`.

**Comment by jancavan** [comment]
@bluefuton yeah, we should. The text looks too wide. I'll 👀 .

**Comment by bluefuton** [comment]
Thanks @jancavan! I've added a class name `conversations__empty-content` so we can target styles.

**Comment by bluefuton** [comment]
Todo: 
- [ ] adjust width and position
- [ ] hide intro banner when showing empty state

**Comment by jancavan** [comment]
I created a new account and I'm seeing my own Pages/Post under Conversations even though I didn't Like or Comment on them:

screenshot 2017-10-11 15 43 42

When I deleted those Posts/Pages, I see the banner and Drake instead of the Conversations empty state:

screenshot 2017-10-11 15 45 42

File: client/reader/stream/index.jsx [hunk 2]
```diff
@@ -484,6 +486,7 @@ class ReaderStream extends React.Component {
 
 				<UpdateNotice count={ this.state.updateCount } onClick={ this.showUpdates } />
 				{ this.props.children }
+				{ showingStream && this.state.posts.length ? this.props.intro : null }
```
Comment (samouri) [inline client/reader/stream/index.jsx]: nit: i prefer the syntax:
```js
{ condition1 && condition2 && thingToRender }
```

File: client/reader/stream/empty.jsx (hunk 1 above)
Comment (samouri) [reply client/reader/stream/empty.jsx]: oooh, so was this button broken for anyone non-english speakers?

**Pull request merged**